weblab/experiments/models.py

```diff
@@ -104,12 +95,12 @@ def latest_version(self):
     @property
     def nice_model_version(self):
         """Use tags to give a nicer representation of the commit id"""
-        return self.model.nice_version(self.model_version)
+        return self.model.nice_version(self.model_version.sha)
```

Comment:
Looks like these are the only places where `nice_version` is called, so it might make more sense to create a `nice_name` method/property on `CachedEntityVersion` so we can just do `return self.model_version.nice_name`? And get rid of `Entity.nice_version`, moving the tests for it into `repocache`?

Reply:
yes thats much cleaner - thanks, done

Reply:
interestingly, I have just seen that there is a call to `experiment.nice_model_version` in `experimentversion_detail.html`

Reply:
Indeed, I don't think we want to get rid of `nice_model_version`.